RELEASE_NOTES

```diff
@@ -1,5 +1,170 @@
 Release Notes
+
+Sherpa 4.8.1
+============
+
+Sherpa 4.8.1 is the standalone counterpart to the 4.8.0 release, which was
+focused on supporting CIAO 4.8. In particular, this version introduces support
+for newer versions of the dependency, along with some feature enhancements,
+bug fixes and additional, more accurate tests.
+
+The newly supported dependencies:
+  - matplotlib v1.5
+  - numpy 1.10 (with and without mkl support)
+  - xspec v12.9.0i (when building from source)
+  - astropy v1.1.2
+  - region library v4.8 (from CIAO 4.8)
+
+Mode details below (infrastructure changes are not shown):
+
+#102: fix issues when writing out FITS files using the `save_pha` and
+`save_table` commands when using the `astropy`/`pyfits` backend (bug #46).
+Fix for when the notice2d_id, notice2d_image, and the ignore version functions
+are called with an invalid identifier (i.e. an identifier that i snot an integer
```

Comment:
There's another typo `i snot an` -> `is not an`. I didn't want to submit a fix for this as I think it worth collecting all the changes and making a limited number of fixes to the document.